Core v2 docs update (#819)
spatialos/gdk-for-unity: docs/content/glossary.md

### Connection

> Before the [worker](#worker) can interact with the [SpatialOS world](#spatialos-world), the worker must connect to SpatialOS. The connection is established by an instance of the `Improbable.Gdk.Core.Worker` class. See [Connecting to SpatialOS]({{urlRoot}}/content/connecting-to-spatialos) for more information.

Reviewer: ```suggestion
Before the [worker](#worker) can interact with the [SpatialOS world](#spatialos-world), the worker must connect to SpatialOS. The connection is established using the `Improbable.Gdk.Core.Worker` class. See [Connecting to SpatialOS]({{urlRoot}}/content/connecting-to-spatialos) for more information.
```

> Also referred to as "synchronization".

> Replication is the process by which all workers in the [SpatialOS world](#spatialos-world) maintain a consistent model of that world. It can be achieved by either [standard](#standard-replication).

Reviewer: ```suggestion
Replication is the process by which all workers in the [SpatialOS world](#spatialos-world) maintain a consistent model of that world. It can be achieved by using the [standard replication](#standard-replication).
```

Author: Not sure if `the` is correct here

Author: Honestly I'd like to remove this entirely given that there is no standard replication or custom replication - just replication
